<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8CNVCLP" failures="2" tests="4" name="PetTests" time="4.688" errors="0" timestamp="2024-08-30T01:24:00 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="2.498"/>
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="0.258">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testGetUserByName(UserTests.java:79)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="api.test.UserTests" name="testUpdateUserByName" time="0.224"/>
  <testcase classname="api.test.UserTests" name="testDeleteUserName" time="0.194">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testDeleteUserName(UserTests.java:103)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserName -->
</testsuite> <!-- PetTests -->
